lighthouse-core/report/html/report-styles.css

```diff
@@ -242,6 +248,10 @@
   cursor: pointer;
 }
 
+.lh-container {
+  word-break: break-word;
```

Comment:
just a warning that i've had break-word challenges in devtools.
there's currently this bit over there: https://cs.chromium.org/chromium/src/third_party/blink/renderer/devtools/front_end/audits2/audits2Panel.css?l=40-42&rcl=312a1d6da6d167ba89c90bd051c5181a66a5184c

let's keep an eye out for how this may trigger issues.